diff --git a/runtime/client.d.ts b/runtime/client.d.ts
index 8cfc8fbb4221ba0c5d9508cac2b428ef50230d32..5f764a3ebe7765aa5a2803080da12e3d1fd83562 100644
--- a/runtime/client.d.ts
+++ b/runtime/client.d.ts
@@ -3178,3 +3178,8 @@ export { Value }
 export declare const warnOnce: (key: string, message: string, ...args: unknown[]) => void;
 
 export { }
+
+export { AnyNullClass } from '@prisma/client-runtime-utils';
+export { DbNullClass } from '@prisma/client-runtime-utils';
+export { JsonNullClass } from '@prisma/client-runtime-utils';
+
diff --git a/runtime/index-browser.d.ts b/runtime/index-browser.d.ts
index ca12789acff83849a2e2d96331ca68dfcd306469..8212c484f8ad10af6fd4cf901b8a1ce9d64280c5 100644
--- a/runtime/index-browser.d.ts
+++ b/runtime/index-browser.d.ts
@@ -85,3 +85,9 @@ declare function validator<C, M extends Exclude<keyof C, `$${string}`>, O extend
 declare function validator<C, M extends Exclude<keyof C, `$${string}`>, O extends keyof C[M] & Operation, P extends keyof Args<C[M], O>>(client: C, model: M, operation: O, prop: P): <S>(select: Exact<S, Args<C[M], O>[P]>) => S;
 
 export { }
+
+
+export { AnyNullClass } from '@prisma/client-runtime-utils';
+export { DbNullClass } from '@prisma/client-runtime-utils';
+export { JsonNullClass } from '@prisma/client-runtime-utils';
+
